hack/build-cross.sh

```diff
@@ -22,7 +22,7 @@ set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
-echo "NOTE: $0 has been replaced by 'make cross'"
+echo "NOTE: $0 has been replaced by 'make cross'. Run 'make help' for help."
```

Comment:
Since these hack scripts are deprecated, I was thinking to add the info to make contributors have a chance to know `make help` if they're still using these scripts. 

I also thought to remove the invocation of the specific command in the script to discourage contributors from using these scripts. `hack/test-integration.sh` has already done this. But I don't know if this will break something, e.g. some contributors might have their own scripts calling these scripts.

@cblecker, WDYT? If it's okay, I will send a new seperate PR to fix this.